src/WooCommerceCheckout.php

```diff
@@ -118,4 +122,34 @@ public static function checkConfirmationEmailField() {
     }
   }
 
+  /**
+   * Check if we are using Amazon V2 in the checkout.
+   *
+   * @return bool
+   */
+  public static function isAmazonPayV2Checkout(): bool {
+    $is_amzon_pay_active = is_plugin_active('woocommerce-gateway-amazon-payments-advanced/woocommerce-gateway-amazon-payments-advanced.php') ?: FALSE;
+    if ($is_amzon_pay_active && isset(WC()->session)) {
```

Comment:
To avoid errors, I am also verifying if the WC()->session is set.

Reply:
This is the actual required fix for my past comment https://github.com/netzstrategen/wordpress-shop-standards/pull/174/files#r765024107